.github/workflows/issue-metrics.yml

```diff
@@ -0,0 +1,40 @@
+name: Monthly issue metrics
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '3 2 1 * *'
+
+permissions:
+  issues: write
+
+jobs:
+  build:
+    name: issue metrics
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get dates for last month
+      shell: bash
+      id: last-month
+      run: |
+        # Calculate the first day of the previous month
+        first_day=$(date -d "last month" +%Y-%m-01)
+
+        # Calculate the last day of the previous month
+        last_day=$(date -d "$first_day +1 month -1 day" +%Y-%m-%d)
+
+        #Set an environment variable with the date range
+        echo "$first_day..$last_day"
+        echo "LAST_MONTH=$first_day..$last_day" >> $GITHUB_OUTPUT
+
+    - name: Run issue-metrics tool
+      uses: github/issue-metrics@v2
+      env:
+        SEARCH_QUERY: 'repo:owner/repo is:issue created: ${{ steps.last-month.outputs.LAST_MONTH }} -reason:"not planned"'
+
+    - name: Create issue
+      uses: peter-evans/create-issue-from-file@v4
+      with:
+        title: Monthly issue metrics report
+        token: ${{ secrets.GITHUB_TOKEN }}
+        content-filepath: ./gh_metrics/issue_metrics.md
```

Comment:
style nit: GitHub seems to complain about newline at the end of the file.

Reply:
weird, added newline and now GH is complaining about new new line :)